charts/bindplane/templates/bindplane.yaml

```diff
@@ -39,6 +39,149 @@ spec:
             - containerPort: 3001
               name: http
           env:
+            - name: BINDPLANE_REMOTE_URL
+              {{- if .Values.config.server_url }}
+              value: {{ .Values.config.server_url }}
```

Comment:
Should we remove config.server_url and use config.remote_url here? Or do we expect the value to include `http` / `https`?

Reply:
So I left this like this for backwards compatibility. In the newer config the old `server_url` is essentially what the new `remote_url` is. I figured while this is technically wrong it would mean a values file you used in an earlier version would still work in the new version. I think when we drop the older logic we can maybe do a check for both values here.